docs/conf/australia/2024/news/tickets-cfp-reminder.rst

```diff
@@ -0,0 +1,72 @@
+:template: {{year}}/generic.html
+
+.. post:: August 6, 2024
+   :tags: {{shortcode}}-{{year}}, website, cfp, tickets
+
+Ticket Sales, Grant Program, and Call for Proposals Reminder
+=============================================================
```

Comment:
I'd prefer this to be in the order of topics (and I think leading with CFP is correct)